Tools/MGCB/ConsoleLogger.cs

```diff
@@ -11,13 +11,13 @@ public class ConsoleLogger : ContentBuildLogger
     {
         public override void LogMessage(string message, params object[] messageArgs)
         {
-			Console.WriteLine(message, messageArgs);
+			Console.WriteLine(IndentString + message, messageArgs);
```

Comment:
I suspect this doesn't handle messages with embedded new line well.  For example when logging an exception from a pipeline error.

To do this right I've done this in the past:

```            
            message = string.Format(message, messageArgs);

            if (message.Contains('\n'))
            {
                var lines = message.Split(new [] { Environment.NewLine }, StringSplitOptions.None);
                foreach (var line in lines)
                {
                    Console.WriteLine(IndentString + line);
                }
                return;
            }

            Console.WriteLine(IndentString + message);
```